lib/GrabViaRestPlugin/src/GrabViaRestPlugin.h

```diff
@@ -316,41 +305,27 @@ class GrabViaRestPlugin : public PluginWithConfig
     mutable MutexRecursive   m_mutex;             /**< Mutex to protect against concurrent access. */
     bool                     m_isConnectionError; /**< Is connection error happened? */
     bool                     m_hasTopicChanged;   /**< Has the topic content changed? */
-
-    /**
-     * Defines the message types, which are necessary for HTTP client/server handling.
-     */
-    enum MsgType
-    {
-        MSG_TYPE_INVALID = 0, /**< Invalid message type. */
-        MSG_TYPE_RSP,         /**< A response, caused by a previous request. */
-        MSG_TYPE_CONN_CLOSED, /**< The connection is closed. */
-        MSG_TYPE_CONN_ERROR   /**< A connection error happened. */
-    };
+    int                      m_restId;

```

Comment:
BTW try not to use int, short int and etc. Use int32_t instead which are platform independent.